javascript/src/main/pages/AnalyzeMessageData/AnalyzeReactions.js

```diff
@@ -1,7 +1,52 @@
-import React from "react";
+import React, {useState} from "react";
+import { useAuth0 } from "@auth0/auth0-react";
+import useSWR from "swr";
+import { fetchWithToken } from "main/utils/fetch";
+import MessageTableReaction from "main/components/ChannelMessageReaction/MessageTableReaction"
+import { Form, Button } from "react-bootstrap";
+
 
 const AnalyzeReactions = () => {
-    return <h1>Analyze Reactions</h1>
+    const { getAccessTokenSilently: getToken } = useAuth0();
+    const [ searchReaction, setSearchReaction ] = useState('');
+    const [ searchResults, setSearchResults ] = useState([]);
+    
+    const handleSearchReactionOnChange = (event) => {
+        setSearchReaction(event.target.value);
+    };
+    
+    const handleSearchReactionOnSubmit = () => {
+        var url = '';
+        if(searchReaction === "+1") {
+            const searchReaction2 = "%2B1";
+            const url1 = `/api/members/messages/reactionsearch?searchReaction=${searchReaction2}`;
+            url = url1;
+        } else {
+            const url2 = `/api/members/messages/reactionsearch?searchReaction=${searchReaction}`;
+            url = url2;
```

Comment:
Is there a reason why we have to define `url1` and `url2` when we can just modify `url` directly?

Reply:
Also, to make this even more concise, consider using the ternary operator